src/app/rxjs/rxjs.component.html

```diff
@@ -1,3 +1,44 @@
-<p>
-  rxjs works!
-</p>
+<div class="container">
+    <mat-card>
+        <div class="banner">
+            <div class="title accent">RxJS</div>
+            <h1 class="rxjs"> Reactive Extensions Library for JavaScript </h1>
+            <a mat-raised-button color="accent" [routerLink]="['installation']">Get started</a>
+        </div>
+    </mat-card>
+
+
+    <div class="tiles">
+        <div class="tile">
+            <h3>The ReactiveX library for JavaScript</h3>
+
+            <p>
+                RxJS is a library for reactive programming using Observables, to make it easier to compose asynchronous or callback-based
+                code. This project is a rewrite of Reactive-Extensions/RxJS with better performance, better modularity, better
```

Comment:
Can you please utilize translation for this content ?

Reply:
sure